data/migrations/versions/909d725887d3_add_composite_index_on_manifestblob.py

```diff
@@ -11,14 +11,19 @@
 down_revision = "88e64904d000"
 
 import sqlalchemy as sa
+from sqlalchemy.engine.reflection import Inspector
 
 
 def upgrade(op, tables, tester):
-    op.create_index(
-        "manifestblob_repository_id_blob_id",
-        "manifestblob",
-        ["repository_id", "blob_id"],
-    )
+    inspector = Inspector.from_engine(op.get_bind())
+    manifestblob_indexes = inspector.get_indexes("manifest")
```

Comment:
I'm getting an error 

```
sqlalchemy.exc.InternalError: (pymysql.err.InternalError) (1061, "Duplicate key name 'manifestblob_repository_id_blob_id'")
[SQL: CREATE INDEX manifestblob_repository_id_blob_id ON manifestblob (repository_id, blob_id)]
(Background on this error at: http://sqlalche.me/e/2j85)
```
The table here should be `manifestblob`

Reply:
Fixed